source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/Dtos/Validation/ValidationRuleIdentifier.cs

```diff
@@ -43,5 +43,6 @@ public enum ValidationRuleIdentifier
         MeteringPointDoesNotExist = 29, // VR200 / E10
         ChargeDoesNotExist = 30, // VR679 / E0I
         ChargeLinkUpdateNotYetSupported = 31, // VR902 / D13
+        SubsequentBundleOperationsFail = 33, // VR??? / D14
```

Comment:
I have added this new rule to our internal excel sheet with validations; the number we can use is 906.
Also suggest name to be changed here to follow Henrik's comment.

```suggestion
        PreviousOperationsMustBeValidRule = 33, // VR906 / D14
```